### DISCUSSION TITLE
请问作者大佬是否能打包debian系发行版的deb包呢？

### DISCUSSION THREAD

**Discussion by ZeaKyX** [wh201906/SerialTest#40] [Q&A] [answered]
如题，~~对不起又是我来刁难大佬了~~
如果能申请发布到什么软件源就更好了

**Comment by wh201906** [comment]
> 如果能申请发布到什么软件源就更好了

有的啊，你看看README里面，有Flathub源和AUR源，debian系的发行版可以用Flathub源安装

**Reply by ZeaKyX** [reply]
Flathub需要代理捏，我看cutecom都进了debian、ubuntu官方源，这个应该也行的吧
目前编译也没有用到deb工具吧

**Reply by ZeaKyX** [reply]
羡慕arch linux用户

**Reply by wh201906** [reply, marked as answer]
这两个软件源都是因为有好心人帮忙才打包上去的，后来我想着Flathub能支持绝大部分的发行版，所以就没针对其它软件源打包了。
考虑到debian系发行版的用户量，也许我有空了也会折腾一下？

**Reply by wh201906** [reply]
linux系统上面跑个proxychains给其它app提供代理还是挺方便的，你可以试试
